usr/share/rear/lib/layout-functions.sh

```diff
@@ -427,6 +444,53 @@ get_partition_number() {
     echo $partition_number
 }
 
+# Extract the underlying device name from the full partition device name.
+# Underlying device may be a disk, a multipath device or other devices that can be partitioned.
+# Should we use the information in $LAYOUT_DEPS, like get_parent_component does,
+# instead of string munging?
+function get_device_from_partition() {
+    local partition_block_device
```

Comment:
@pcahyna 
thank you for the separated function to get the underlying device from a partition device.
It cleans up the code and makes it possible to easily improve the current
implementation (i.e. string munging) by something better in the future if needed.